src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,74 @@
+package duke.parser;
+
+import duke.commands.*;
+import duke.data.exception.DukeException;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.text.ParseException;
+
+public class ParserTest {
+
+    /** Tests list feature of Parser */
+    @Test
+    public void parse_list() throws DukeException, ParseException {
```

Comment:
Same for these methods, wondering if test method names should be in camelCase as well?